DotnetCLIToolsVersion.txt

```diff
@@ -1 +1 @@
- 2.1.403
+2.1.403
```

Comment:
More a general observation unrelated to this pr - shouldn't this be in a file named global.json?

Reply:
That is an old piece of machinery for grabbing a dotnet cli, it came from the original dotnet coreclr build tools.  Eventually it will be replaced with something for getting a LKG.  since that is what it will eventually become.